saleor/dashboard/group/views.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import unicode_literals
+
+from django.shortcuts import redirect, get_object_or_404
+from django.contrib import messages
+from django.contrib.auth.models import Group
+from django.template.response import TemplateResponse
+from django.utils.translation import pgettext_lazy
+
+from .forms import GroupPermissionsForm
+from ..views import staff_member_required
+
+
+@staff_member_required
+def groups_list(request):
```

Comment:
We could name this view `group_list` to be consistent with rest of the views in this file.

Reply:
yea i was changin those names to non plural, missed one place